Sky Atlas/Sky Atlas.html

```diff
@@ -6930,225 +6930,18 @@ <h1>Sections & Primary Docs</h1>
         <tr>
           <td>
             <dfn>A.3.5
-              - A1 - Surplus Buffer And Smart Burn Engine - Surplus Buffer Upper Limit</dfn>
+              - A1 - Surplus Buffer And Smart Burn Engine - Surplus Buffer Splitter Parameters</dfn>
           </td>
-          <td>Surplus Buffer And Smart Burn Engine - Surplus Buffer Upper Limit</td>
+          <td>Surplus Buffer And Smart Burn Engine - Surplus Buffer Splitter Parameters</td>
           <td>Section</td>
-          <td>This Section defines the Surplus Buffer Upper Limit and key parameters of the Smart Burn Engine.
+          <td>The current Surplus Buffer Splitter parameters are:• splitter.hump: 60m (Threshold of Surplus buffer for Splitter to activate)•
+              Estimated Net protocol Income: 100,765,523• 50% of estimated net protocol income is sent to Splitter: 50,382,762 (for SBE and 
+              Seal Module, remaining 50% is accumulated in Surplus Buffer). This value must be regularly updated through an executive vote to 
+              continue to target 50% of the estimated net protocol income.• splitter.bump: 25,000 DAI• splitter.hop: 15,649 seconds• 70% of 
```

Comment:
bump and hump are vow parameters.
```suggestion
          <td>The current Surplus Buffer Splitter parameters are:• vow.hump: 60m (Threshold of Surplus buffer for Splitter to activate)•
              Estimated Net protocol Income: 100,765,523• 50% of estimated net protocol income is sent to Splitter: 50,382,762 (for SBE and 
              Seal Module, remaining 50% is accumulated in Surplus Buffer). This value must be regularly updated through an executive vote to 
              continue to target 50% of the estimated net protocol income.• vow.bump: 25,000 DAI• splitter.hop: 15,649 seconds• 70% of 
```